src/main/scala/temple/generate/server/go/service/dao/GoServiceDAOFunctionsGenerator.scala

```diff
@@ -29,8 +28,8 @@ object GoServiceDAOFunctionsGenerator {
       operation match {
         case List =>
           root.createdByAttribute match {
```

Comment:
Again could use the `.contains` approach from above, but whatever you think is cleaner